src/librustc_borrowck/borrowck/mir/mod.rs

```diff
@@ -43,8 +43,10 @@ fn has_rustc_mir_with(attrs: &[ast::Attribute], name: &str) -> Option<P<MetaItem
         if attr.check_name("rustc_mir") {
             let items = attr.meta_item_list();
             for item in items.iter().flat_map(|l| l.iter()) {
-                if item.check_name(name) {
-                    return Some(item.clone())
+                if let Some(mi) = item.meta_item() {
+                    if mi.check_name(name) {
+                        return Some(mi.clone())
+                    }
```

Comment:
Actually, we can call check_name directly on the list item. I'll change it to that.